wordpress-plugins/kunta-api-management/embeds.php

```diff
@@ -0,0 +1,61 @@
+<?php
+  defined( 'ABSPATH' ) || die( 'No script kiddies please!' );
+
+  require_once ('vendor/autoload.php');
+  require_once ('kunta-api-client.php');
+  require_once ('service-loader.php');
+  
+  use Sunra\PhpSimple\HtmlDomParser;
+  
+  if (is_admin()) {
+  	add_filter('content_edit_pre', 'kunta_api_setup_embeds');
+  }
+
+  add_filter('the_content', 'kunta_api_content_filter');
+
+  function kunta_api_setup_embeds($content) {
+    $serviceLoader = new ServiceLoader();
+  	$dom = HtmlDomParser::str_get_html($content);
+    
+  	foreach ($dom->find('*[data-type="kunta-api-embedded-service-component"]') as $article) {
+      $serviceId = $article->{'data-service-id'};
+      $serviceComponent = $article->{'data-service-component'};
+      $article->class = 'mceNonEditable';
+      $article->title = __( 'Embedded read-only content from PTV', 'kunta_api_management');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "kunta_api_management" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=php%3AS1192)